src/js/component.js

```diff
@@ -684,8 +684,8 @@ class Component {
    * @param  {Event|Object|String} event  A string (the type) or an event object with a type attribute
    * @return {Component}       self
    */
-  trigger(event) {
-    Events.trigger(this.el_, event);
+  trigger(event, ...data) {
```

Comment:
Yeah, we're strictly following the Airbnb guide now which notes that.
https://github.com/airbnb/javascript#events